cmd/envtool/main.go

```diff
@@ -219,7 +198,39 @@ func setupMonilaAndValues(ctx context.Context, pgPool *pgdb.Pool) {
 	logger.Infof("Done in %s.", time.Since(start))
 }
 
-func main() {
+//nolint:forbidigo // Printf used to make diagnostic data easier to copy.
+func printDiagnosticData(err error) {
+	info := version.Get()
+	fmt.Printf(`Looks like something went wrong..
+Please file an issue with all that information below:
+	Go version: %s
+	OS: %s
+	Arch: %s
+	Version: %s
+	Commit: %s
+	Branch: %s
+	Error: %v
```

Comment:
I think we can move to a separate block – it is significantly different from other lines

Reply:
Fixed.